zerver/lib/test_helpers.py

```diff
@@ -128,33 +132,39 @@ def cursor_executemany(self, sql, params=()):
 
 
 def find_key_by_email(address):
+    # type: (str) -> str
     from django.core.mail import outbox
     key_regex = re.compile("accounts/do_confirm/([a-f0-9]{40})>")
     for message in reversed(outbox):
         if address in message.to:
             return key_regex.search(message.body).groups()[0]
 
 def message_ids(result):
+    # type: (Dict[str, Any]) -> Set[str]
     return set(message['id'] for message in result['messages'])
```

Comment:
I think that's because the files that call this function (mostly zerver/tests/test_messages.py) are in the mypy exclude list, so there's nothing for it to check this annotation for consistency with :)